examples/bindeps/flake.nix

```diff
@@ -0,0 +1,50 @@
+{
+  description = "Building packages with bindeps";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+
+    crane = {
+      url = "github:ipetkov/crane";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+
+    flake-utils.url = "github:numtide/flake-utils";
+
+    rust-overlay = {
+      url = "github:oxalica/rust-overlay";
+      inputs = {
+        nixpkgs.follows = "nixpkgs";
+        flake-utils.follows = "flake-utils";
+      };
+    };
+  };
+
+  outputs = { nixpkgs, crane, flake-utils, rust-overlay, ... }:
```

Comment:
Do you mind turning this example into a "regular" flake check test?

There isn't anything specific that consumers have to change about their `crane` invocations for bindeps (besides making sure their toolchain supports the necessary targets), so we probably don't need to make a specific template for it. Though I do want to keep it as a test!

Reply:
That's exactly what I wanted to do initially, but it would require a custom toolchain and I have not found an example check using a custom toolchain. But I'll work on that

Reply:
We have `rust-overlay` available as an input, actually the `noStd` test uses a custom toolchain if you want to use that as a starting point: https://github.com/ipetkov/crane/blob/105e27adb70a9890986b6d543a67761cbc1964a2/checks/default.nix#L237-L251

Reply:
Right, that was the test case I added myself a while ago :laughing: 097afac